molecule/provisioners/dockerprovisioner.py

```diff
@@ -113,7 +113,7 @@ def build_image(self):
             dockerfile = '''
             FROM {}:{}
             RUN bash -c 'if [ -x "$(command -v apt-get)" ]; then apt-get update && apt-get install -y python sudo; fi'
-            RUN bash -c 'if [ -x "$(command -v yum)" ]; then yum makecache fast && yum update -y && yum install -y python sudo; fi'  # NOQA
+            RUN bash -c 'if [ -x "$(command -v yum)" ]; then yum makecache fast && yum update -y && yum install -y python sudo; fi'
 
```

Comment:
That's a NOQA that I added and then removed because it was in the context of the line. There's already a noqa in there, though it doesn't work for me.